internal/remotewrite/config.go

```diff
@@ -9,6 +9,9 @@ import (
 	"github.com/newrelic/newrelic-prometheus-configurator/internal/promcfg"
 )
 
+// Name identifies the newrelic remote_write entry used in logs and prometheus self metrics.
+const Name = "newrelic_conf"
```

Comment:
Nit: What about something like `newrelic_remote_write` or similar?

Reply:
this is used to identify the remote name in metrics and logs here is an example
`prometheus_remote_storage_shards_max{remote_name="newrelic_conf",url="htt...`
I used `newrelic` at first but then thought it could probably be used by any customer sending to rw and thought that a name more unique to the configurator would be nice in case we want it for identification propouses. 

But yes `newrelic_conf` is not so nice. Perhaps `newrelic_rw` could less redundant and more custom to this solution i guess.